conf/modules.config

```
@@ -190,15 +190,15 @@ process {
     }
 
     withName: GSEA_GSEA {
-        ext.prefix = { "${meta.id}." }
+        ext.prefix = { "${meta.id}.${meta.gmt_name}." }
         publishDir = [
             [
-                path: { "${params.outdir}/tables/gsea/${meta.id}" },
+                path: { "${params.outdir}/tables/gsea/${meta.id}/${meta.gmt_name}" },
```

```suggestion
                path: { "${params.outdir}/tables/gsea/${meta.id}/${gene_sets.baseName}" },
```
I don't think we actually need to go through the meta for this